### PR TITLE
Updated README and other files

### PR DESCRIPTION
**Pull request by ntallar**
## Description

Changes included:
- Updated README for created packages, removed CHANGELOG and CODEOWNERS
- Fixed using 2*k for consistency check instead of k

### CODE REVIEW

File: blockchain-importer/README.md [hunk 1]
```diff
@@ -30,28 +38,19 @@ stack exec cardano-blockchain-importer-mock
...
-## Sockets
-
-`CORS` requests to connect `socket` server are currently restricted to following resources:
-* https://cardano-blockchain-importer.com
-* https://blockchain-importer.iohkdev.io
-* http://cardano-blockchain-importer.cardano-mainnet.iohk.io
-* http://localhost:3100
-
-Change `CORS` policies in `src/Pos/BlockchainImporter/Socket/App.hs` whenever you have to add more resources.
+- Run `/scripts/clean/db.sh` to do a clean synchronization, so that BlockchainImporter will sync and download blockchain from start. Create a
```
Comment (mirkoAlic) [inline blockchain-importer/README.md]: Create a ...?

File: blockchain-importer/README.md [hunk 2]
```diff
@@ -30,28 +38,19 @@ stack exec cardano-blockchain-importer-mock
...
-
-`CORS` requests to connect `socket` server are currently restricted to following resources:
-* https://cardano-blockchain-importer.com
-* https://blockchain-importer.iohkdev.io
-* http://cardano-blockchain-importer.cardano-mainnet.iohk.io
-* http://localhost:3100
-
-Change `CORS` policies in `src/Pos/BlockchainImporter/Socket/App.hs` whenever you have to add more resources.
+- Run `/scripts/clean/db.sh` to do a clean synchronization, so that BlockchainImporter will sync and download blockchain from start. Create a 
+- Connect to cluster as described in  `docs/how-to/connect-to-cluster.md` **FIXME: Add usage of nix for building, currently doesn't work**
```
Comment (mirkoAlic) [inline blockchain-importer/README.md]: There is a task for fix this?

Comment (ntallar) [reply blockchain-importer/README.md]: Created task https://iohk.myjetbrains.com/youtrack/issue/PI-303

**Review by mirkoAlic — APPROVED** [review]
:+1:

**Pull request merged**